File: MyApplication3/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });
        String tk="than";
        String mk="123";
        EditText V1=findViewById(R.id.v1);
        EditText V2=findViewById(R.id.v2);
        Button Btn1= findViewById(R.id.btn1);
        Btn1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
               String ab= V1.getText().toString().trim();
               String vc= V2.getText().toString().trim() ;
                if (ab.equals(tk) && vc.equals(mk)){
                    Toast.makeText(MainActivity.this,"thành công",Toast.LENGTH_SHORT).show();
                }
                else {
                    Toast.makeText(MainActivity.this,"ko thành công",Toast.LENGTH_SHORT).show();

                }
            }
        });
    }
}

File: Tinhtong/app/src/main/java/voquangthan64132137/tinhtong/MainActivity.kt
package voquangthan64132137.tinhtong

import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }
        val Inputa= findViewById<EditText>(R.id.inputa)
        val Inputb= findViewById<EditText>(R.id.inputb)
        val Button1= findViewById<Button>(R.id.btn1)
        val Result= findViewById<TextView>(R.id.result)
        Button1.setOnClickListener{
            TinhTong(Inputa,Inputb,Result)

        }

    }
    private  fun TinhTong(Inputa:EditText,Inputb:EditText,Result:TextView){
        val a=Inputa.text.toString().toInt()
        val b=Inputb.text.toString().toInt()
        val sum=a+b;
        Result.text="kết quả là : $sum"
    }
}